### PR TITLE
android studio canary

### PR DESCRIPTION
**Pull request by kevincianfarini**
### Please confirm your submission meets all the criteria

- [ ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
the app-id and the manifest + app-data do not match.

Wouldn't that be easier to just make this the `beta` branch of the originating package?

**Comment by kevincianfarini** [comment]
@hfiguiere what options do we have for distribution versions? Canary !== beta. Canary is essentially rolling release alpha.

File: com.google.AndroidStudioCanary.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.google.AndroidStudio.desktop</id>
```
Comment (bilelmoussaoui) [inline com.google.AndroidStudioCanary.appdata.xml:3]:
```suggestion
	<id>com.google.AndroidStudio</id>
```
the .desktop suffix here is not needed

Comment (kevincianfarini) [reply com.google.AndroidStudioCanary.appdata.xml]: Thanks! Resolved. I wonder why it's present from the regular [Android Studio repo](https://github.com/flathub/com.google.AndroidStudio/blob/master/com.google.AndroidStudio.appdata.xml#L3)? This repo is directly forked from that.

**Comment by kevincianfarini** [comment]
@hfiguiere After thinking more, I guess `beta` being canary doesn't matter. Is there documentation around how beta branches of repos are distributed? I'd ideally like to close this PR and open one against the `com.google.AndroidStudio` repo.

File: com.google.AndroidStudioCanary.appdata.xml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.google.AndroidStudioCanary.appdata.xml]: People tend to confuse the appstream ID/Application ID & the desktop file name.

**Comment by kevincianfarini** [comment]
Closed via [com.google.AndroidStudio beta release](https://github.com/flathub/com.google.AndroidStudio/pull/108).

**Pull request closed without merging**